gravitee-node-notifier/src/main/java/io/gravitee/node/notifier/trigger/NotificationTrigger.java

```diff
@@ -186,10 +194,7 @@ public void handle(Long event) {
                                                             error
                                                         )
                                                     )
-                                                    .doFinally(() ->
-                                                        //trigger a new attempt
-                                                        this.scheduleNextAttempt()
-                                                    )
+                                                    .doFinally(this::scheduleNextAttempt)
```

Comment:
I know that it's not this PR's main objective, but I can see here that, in case of an error, the next attempt will not be re-triggered. There is a doOnError that properly logs the error but the error is still propagated. Wouldn't an `onErrorResumeNext()` be more appropriate?

Reply:
Thanks 👍 
Will look into it.